app/assets/javascripts/modules/hide-other-links.js

```diff
@@ -51,7 +51,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
     this.showLink.classList.add('show-other-content', 'govuk-link')
     this.showLink.innerHTML = linkText
-    this.showLink.href = '#'
+    this.showLink.setAttribute('aria-expanded', 'false')
+    this.showLink.setAttribute('aria-controls', 'other-content')
```

Comment:
Thanks -  though `aria-controls` needs an element ID, rather than a CSS class

Reply:
Ah, I forgot to add the ID assignment before pushing that up. Fixed now, at line 75.

Reply:
@AshGDS I forgot to tag you, if you could re-check the above that would be great thanks.